dfget/core/downloader/p2p_downloader/p2p_downloader.go

```diff
@@ -280,15 +286,9 @@ func (p2p *P2PDownloader) pullPieceTask(item *Piece) (
 			break
 		}
 
-		sleepTime := time.Duration(rand.Intn(p2p.maxTimeout-p2p.minTimeout)+p2p.minTimeout) * time.Millisecond
-		logrus.Infof("pull piece task(%+v) result:%s and sleep %.3fs", item, res, sleepTime.Seconds())
-		time.Sleep(sleepTime)
-
-		// gradually increase the sleep time, up to [800-1600]
-		if p2p.minTimeout < 800 {
-			p2p.minTimeout *= 2
-			p2p.maxTimeout *= 2
-		}
+		actual, expected := p2p.sleepInterval()
+		logrus.Infof("pull piece task(%+v) result:%s and sleep actual:%.3fs expected:%.3fs",
```

Comment:
Is it necessary to record expected sleep interval which is actually a random value.

Reply:
Yeah, it's more like a `debug` log, but I think it's necessary to output it in `info` level if `actual > expected`.